### PR TITLE
Change the numa_cell xml action according numa_cell change in vm_xml

### PR DESCRIPTION
**Pull request by jinqi7**
Signed-off-by: qijing <jinqi@redhat.com>

### CODE REVIEW

**Comment by jinqi7** [comment]
Related to avocado-vt PR https://github.com/avocado-framework/avocado-vt/pull/2694

**Comment by jinqi7** [comment]
avocado run --vt-type libvirt nvdimm.back_file.label
/root/.libvirt-ci-venv-ci-runtest-U0VKBp/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : bab70a64ea6d8c0689cf4c373c2ce35703d67db4
JOB LOG    : /root/avocado/job-results/job-2020-08-06T05.00-bab70a6/job.log
 (1/3) type_specific.io-github-autotest-libvirt.nvdimm.back_file.label.default: PASS (75.33 s)
 (2/3) type_specific.io-github-autotest-libvirt.nvdimm.back_file.label.check_life_cycle: PASS (249.63 s)
 (3/3) type_specific.io-github-autotest-libvirt.nvdimm.back_file.label.discard: PASS (5.33 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
 avocado run --vt-type libvirt libvirt_mem.positive_test.mem_basic
/root/.libvirt-ci-venv-ci-runtest-U0VKBp/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : f3e1d68c1c000b0fa7b7b3cdf56122f098681ed9
JOB LOG    : /root/avocado/job-results/job-2020-08-06T04.32-f3e1d68/job.log
 (1/3) type_specific.io-github-autotest-libvirt.libvirt_mem.positive_test.mem_basic.default: PASS (60.41 s)
 (2/3) type_specific.io-github-autotest-libvirt.libvirt_mem.positive_test.mem_basic.cold_plug_discard: PASS (66.57 s)
 (3/3) type_specific.io-github-autotest-libvirt.libvirt_mem.positive_test.mem_basic.discard: PASS (96.54 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 226.05 s
avocado run --vt-type libvirt guest_numa.possitive_test.no_hugepage.no_mem_backend.topology.no_numatune_memnode.numatune_mem
/root/.libvirt-ci-venv-ci-runtest-U0VKBp/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 0632b3828204fdff002a92c8873fed151b39aedd
JOB LOG    : /root/avocado/job-results/job-2020-08-06T04.47-0632b38/job.log
 (1/1) type_specific.io-github-autotest-libvirt.guest_numa.possitive_test.no_hugepage.no_mem_backend.topology.no_numatune_memnode.numatune_mem: PASS: 1 in nodeset out of range (88.46 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 100.84 s

File: libvirt/tests/src/migration/migrate_mem.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        libvirt.set_vm_disk(vm, params)
+
+        # Update libvirtd configuration
+        if libvirtd_conf_dict:
+            if os.path.exists(log_file):
+                logging.debug("Delete local libvirt log file '%s'", log_file)
+                os.remove(log_file)
+            logging.debug("Update libvirtd configuration file")
+            conf_type = params.get("conf_type", "libvirtd")
+            if conf_type == "libvirtd" and utils_split_daemons.is_modular_daemon():
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_mem.py:112]: I assume these can be simplified by https://github.com/avocado-framework/avocado-vt/pull/2730

File: libvirt/tests/src/migration/migrate_mem.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    remote_virsh_dargs = {'remote_ip': remote_ip, 'remote_user': remote_user,
+                          'remote_pwd': remote_pwd, 'unprivileged_user': None,
+                          'ssh_remote_auth': True}
+
+    migration_test = migration.MigrationTest()
+    migration_test.check_parameters(params)
+
+    # Params for NFS shared storage
+    shared_storage = params.get("migrate_shared_storage", "")
+    if shared_storage == "":
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_mem.py:43]: I assume this is not necessary since it is will be initialized in  base.cfg under avocado-vt

**Pull request closed without merging**